src/frontend/src/api/modules/feed.js

```diff
@@ -0,0 +1,14 @@
+import ApiService from '@/api';
+
+const BASE_URL = '/articles';
```

Comment:
```suggestion
const BASE_URL = '/api/articles';
```
서버쪽 URL이 변경되었습니다...!

Reply:
이 부분 반영을 하지 않았네요!! 반영하겠습니다 😢

Reply:
이 부분은 src/api/index.js에 있는 PREFIX가 반영됩니다!! 그래서 api는 붙어서 보내져요ㅎㅎ
다만 프록시 설정을 지우지 않아서, 그 부분 수정하겠습니다.